acceptance/setup/aio/pre-suite/015_PackageHostsPresets.rb

```diff
@@ -1 +1,9 @@
-master['use-service'] = true
+if options['is_puppetserver']
+  master['use-service'] = true
+  master['puppetservice'] = 'puppetserver'
+  master['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
k, let me rejigger these commits, because there are some things in here we need in the other PR. As much as I don't want a different rake task, I think ultimately it will be better if we need to add passenger specific setup tasks (which I know we'll need), and if we want to restrict the set of tests that we run in passenger (something we'd like to do).